library/core/api/src/main/kotlin/de/lise/fluxflow/api/versioning/SimpleVersion.kt

```diff
@@ -0,0 +1,19 @@
+package de.lise.fluxflow.api.versioning
+
+/**
+ * This is the most basic [Version] implementation, that only compares the string representations returned by
+ * [version].
+ * 1. If they match, [VersionCompatibility.Compatible] will be returned
+ * 2. If they don't match, [mismatchCompatibility] will be returned.
```

Comment:
oder eher `[VersionCompatibility.Unknown]`?

Reply:
@bobmazy Do you want me to 
a) remove the configurability, so the implementation always returns `Unknown`
b) clarify within the KDoc, that the `mismatchCompatibility` is set to `Unknown` by default?

Reply:
@masinger b)

I wanted to clarify whether it would be better to mention the default value of `mismatchCompatibility` directly in the KDoc or to highlight this variable only. 
It would also be interesting to know if IntelliJ displays the default value of the variable automatically.

Personally, I now think your current approach is better for handling changes to the default value.

Reply:
I adapted the KDoc and explicitly mentioned the default value.